packages/amplify-e2e-tests/src/__tests__/api_8.test.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  addApiWithoutSchema,
+  addFeatureFlag,
+  addFunction,
+  addRestApi,
+  amplifyPushGraphQlWithCognitoPrompt,
+  amplifyPushUpdate,
+  checkIfBucketExists,
+  createNewProjectDir,
+  deleteProject,
+  deleteProjectDir,
+  getAppSyncApi,
+  getProjectMeta,
+  initJSProjectWithProfile,
+  updateApiSchema,
+} from 'amplify-e2e-core';
+import { readdirSync, readFileSync } from 'fs';
+import * as path from 'path';
+
+// to deal with bug in cognito-identity-js
+(global as any).fetch = require('node-fetch');
+// to deal with subscriptions in node env
+(global as any).WebSocket = require('ws');
+
+describe('amplify add api (REST)', () => {
```

Comment:
nit: Not all tests in this block are for REST

Reply:
Updated description.